notifications-worker.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/dxe/alc-mobile-api/expo"
+
```

Comment:
nit: remove the blank lines between the github.com imports; typically you just have two import blocks: the standard library and non-standard packages